### PR TITLE
TEC-658, add blog identities

### PR DESCRIPTION
**Pull request by gcedo**
Lives @ http://gcedo.github.io/component-header/

Starting from the comments in #20 by @keithamus, this is how I approached this feature implementation.

It wasn't possible to use the same image for both the mobile and desktop version of the banner, as they do differ in some cases

screen shot 2016-03-23 at 09 27 38

screen shot 2016-03-23 at 09 27 53

Therefore I went for a different approach than the one suggested by @keithamus:

1. **desktop** svgs, background removed, cropped and optimised, reside in `banners`
2. **mobile** svgs, background removed, cropped and optimised, reside in `banners-mobile`
3. `scripts/compile-backgrounds` produces one rule for each banner, written as follows:
 ```css
 .banner--bagehot {  background: url("inline-svg");  }

@media (max-width: 36rem) {
        .banner--bagehot {  background: url("inline-svg-mobile");  }
}
 ```
4. The generated css files reside in `backgrounds`, and are imported from `blogCustomisations.css`

### CODE REVIEW

File: src/blogCustomisation.css [hunk 1]
```diff
@@ -1,49 +1,74 @@
...
+}
 
-.header--blog21003976 {
-  background-color: #004388;
-  background-position: 100% 0;
-  background-size:
-    calc(var(--header-image-aspect-ratio, 4.51) * var(--header-viewport-big-height, 300px))
-    var(--header-viewport-big-height, 300px);
+.header--blog-freeexchange {
+  background-color: var(--color-freeexchange);
```
Comment (keithamus) [inline src/blogCustomisation.css]: Variables for the header should start with `header` - so `--header-background-color-freeexchange` would be better here, IMO.

**Comment by keithamus** [comment]
Still not convinced you need both desktop and mobile versions of the SVGs, as an example; the only difference between https://github.com/gcedo/component-header/blob/TEC-658/enhanced-blog-identity/banners-mobile/buttonwood-mobile.svg and https://github.com/gcedo/component-header/blob/TEC-658/enhanced-blog-identity/banners/buttonwood.svg is the size - which could be addressed by `background-size` css.

**Comment by gcedo** [comment]
It's not only the size, the left part of the tree is cropped on mobile. Also, I have tried the approach you suggested, the issue was that there was not a _magic number_ you could apply to every `background-size`, but tinker with each blog banner individually. If you think anyways that that is anyway the best approach, would it be ok for you to open an issue for the next sprint and have it assigned to me?

**Comment by keithamus** [comment]
You can use negative background-position to create a crop. Anyway, I am happy with this, but would like someone else to look. @umbobabo?

**Comment by umbobabo** [comment]
Happy to merge but I will appreciate if you could, as discussed in a previous meeting, remove the "dangerous" component-typography from the dependency change default values as hardcoded values equal to the one provided by the typography.
If you don't want do that now, please open an issue on this component.

**Comment by gcedo** [comment]
#25 created.

**Pull request merged**